applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_incompressible_potential_element.cpp

```diff
@@ -261,6 +272,28 @@ namespace Kratos {
       }
     }
 
+    // Checks the function GetWakeDistances
+    KRATOS_TEST_CASE_IN_SUITE(GetWakeDistances, CompressiblePotentialApplicationFastSuite)
+    {
+      Model this_model;
+      ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+      GenerateElement(model_part);
+      Element::Pointer pElement = model_part.pGetElement(1);
+      pElement->SetValue(WAKE, true);
+
+      BoundedVector<double,3> distances = AssignDistances();
+      pElement->SetValue(ELEMENTAL_DISTANCES, distances);
+
+      const auto returned_distances = PotentialFlowUtilities::GetWakeDistances<2, 3>(*pElement);
+
+      std::vector<double> reference({1.0, -1.0, -1.0});
```

Comment:
Since we know 100% the size of the references vector, we can use BoundedVector or std::arrays for better memory allocation (we can do a clean up of the cpp tests in a new PR). I used std::arrays in the [compressible cpp tests.](https://github.com/KratosMultiphysics/Kratos/blob/1812c00992507656a6ada4929dbdbb5ca15d42b2/applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_compressible_potential_element.cpp#L166)

Reply:
Sounds good! :+1: